src/DynamoCore/Core/DynamoLoader.cs

```diff
@@ -65,10 +65,12 @@ private void OnAssemblyLoaded(Assembly assem)
         /// the bin/nodes directory. Add the types to the searchviewmodel and
```

Comment:
Additional `nodes` directories are now being passed to `DynamoLoader` instead of it having to depend on some `static` list.